en/vespa8-release-notes.html

```diff
@@ -476,22 +476,29 @@ <h3 id="removed-java-classes-and-methods">Removed Java Classes and methods</h3>
   The following classes are no longer public API and have been moved to Vespa internal packages:
 </p>
 <table class="table">
-  <thead><tr><th>Package</th><th>Class</th></tr></thead>
+  <thead><tr><th>Package</th><th>Class</th><th>Migration advice</th></tr></thead>
   <tbody>
   <tr>
     <td><em>com.yahoo.config.subscription</em></td>
     <td>All classes, except
       <a href="https://javadoc.io/doc/com.yahoo.vespa/config/latest/com/yahoo/config/subscription/ConfigGetter.html"><em>ConfigGetter</em></a></td>
+    <td>Config should be injected via the constructor of your component class.</td>
```

Comment:
maybe link to https://docs.vespa.ai/en/jdisc/injecting-components.html ?

Reply:
Thanks, fixed.